htdocs/admin/dict.php

```diff
@@ -2015,7 +2015,7 @@ function fieldList($fieldlist, $obj = '', $tabname = '', $context = '')
 			print '<td class="'.$classtd.'">';
 			$transfound = 0;
 			$transkey = '';
-			if (in_array($fieldlist[$field], array('label', 'libelle')))		// For label
+			if (in_array($fieldlist[$field], array('label', 'libelle')) and !empty($obj->code))		// For label
 			{
 				// Special case for labels
 				if ($tabname == MAIN_DB_PREFIX.'c_civility') {
```

Comment:
I think test 
`&& !empty($obj->code)`
should be added here instead.

Reply:
Yes but all label translations are based on `$obj->code`. So we can have the same issue with other dictionary.

Reply:
@eldy do I have to review my PR ?

Reply:
Yes, but for some other $obj->code may simply not exists.
So test should be added on the 2 that are concerned, so the 2 with the if ($tabname == ...

Reply:
Ok, I changed it.